qutebrowser/test/mainwindow/statusbar/test_progress.py

```diff
@@ -0,0 +1,74 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2014-2015 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""Test Percentage widget."""
+
+from collections import namedtuple
+
+import pytest
+
+from qutebrowser.browser import webview
+from qutebrowser.mainwindow.statusbar.progress import Progress
+
+
+@pytest.fixture
+def progress_widget(qtbot, default_config):
```

Comment:
Example of a fixture. In pytest all a test function has to do to use a fixture is to declare it as a parameter. As can be seen here, fixtures can also use other fixtures in the same manner. [Reference](http://pytest.org/latest/fixture.html).

Also in this example, I also make sure that the initial state of the object is correct.